mosip-acceptance-tests/ivv-orchestrator/Dockerfile

```diff
@@ -8,51 +7,33 @@ LABEL source=${SOURCE}
 LABEL commit_hash=${COMMIT_HASH}
 LABEL commit_id=${COMMIT_ID}
 LABEL build_time=${BUILD_TIME}
-
 # can be passed during Docker build as build time environment for github branch to pickup configuration from.
 ARG container_user=mosip
-
-# can be passed during Docker build as build time environment for github branch to pickup configuration from.
 ARG container_user_group=mosip
-
-# can be passed during Docker build as build time environment for github branch to pickup configuration from.
 ARG container_user_uid=1001
-
-# can be passed during Docker build as build time environment for github branch to pickup configuration from.
 ARG container_user_gid=1001
-
 # set working directory for the user
 ENV work_dir=/home/${container_user}/
-
-ARG KUBECTL_VERSION=1.22.9
-
 # install packages and create user
-RUN apt-get update \
-    && apt install jq -y \
-    && groupadd -g ${container_user_gid} ${container_user_group} \
+RUN groupadd -g ${container_user_gid} ${container_user_group} \
     && useradd -u ${container_user_uid} -g ${container_user_group} -s /bin/bash -m ${container_user} -d ${work_dir} \
-    && curl -LO "https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl" \
-    && chmod +x kubectl \
-    && mv kubectl /usr/local/bin/
-
+    && chown -R ${container_user}:${container_user} /home/${container_user}
```

Comment:
please revert this change as the changes are previous changes already satisfied

Reply:
this changes are reverted